documentation/endpoints/signups.md

```diff
@@ -119,85 +119,64 @@ Example request body:
 ]
 ```
 
-Response: The signup and associated event and posts.
+Response: The signup and associated events and posts.
 
 Example response:
 ```
 {
   "data": {
-    "id": 160,
-    "northstar_id": "5639066ba59dbfe6598b4567",
+    "signup_id": 160,
+    "signup_event_id": 799,
+    "submission_type": "user",
+    "northstar_id": "1234",
     "campaign_id": "6",
     "campaign_run_id": "1740",
     "quantity": null,
     "quantity_pending": "200",
     "why_participated": "bcuz I luv endpointz",
+    "signup_source": "phoenix-web",
     "created_at": "2017-01-19T19:11:14+0000",
     "updated_at": "2017-01-19T19:11:14+0000",
-    "post": {
+    "posts": {
       "data": [
         {
-          "signup_id": 160,
-          "northstar_id": "5639066ba59dbfe6598b4567",
-          "campaign_id": 6,
-          "campaign_run_id": 1740,
+          "postable_id": 16,
+          "post_event_id": 29,
+          "submission_type": "user",
+          "postable_type": "Rogue\\Models\\Photo",
           "content": {
-            "type": "post_photo",
             "media": {
-              "url": "https://s3.amazonaws.com/ds-rogue-qa/uploads/reportback-items/160-1484853074.jpeg",
-              "edited_url": null
+              "url": "https://s3.amazonaws.com/ds-rogue-prod/uploads/reportback-items/1334-1484771619.jpeg",
+              "edited_url": "https://s3.amazonaws.com/ds-rogue-prod/uploads/reportback-items/edited_1334-1484771621.jpeg"
             },
-            "caption": "LOOK AT THAT CAT",
+            "caption": "caption",
             "status": "pending",
-            "created_at": "2017-01-19T19:11:15+0000",
-            "updated_at": "2017-01-19T19:11:15+0000"
-          },
-          "event": {
-            "data": {
-              "event_id": "290",
-              "event_type": "post_photo",
-              "submission_type": "",
-              "created_at": "2017-01-19T19:11:14+0000",
-              "updated_at": "2017-01-19T19:11:14+0000"
-            }
+            "remote_addr": null,
+            "post_source": null,
+            "created_at": "2017-01-18T20:33:41+0000",
+            "updated_at": "2017-01-18T20:33:41+0000"
           }
         },
+      ],
+    },
+    "events": {
```

Comment:
cool! I was trying to figure out if any of this will change anything for the migration, but we just need to grab `signup_id` and then the event id for the posts, and I thiiiink this actually makes it easier to do so!